### PR TITLE
Improve Windows Chef Infra Client Docs

### PR DESCRIPTION
**Pull request by tas50**
Link to the actual downloads page
Remove comparisons to running Infra Client as a service. We're not selling them on tasks anymore. We're just telling them how it is
Link to the chef_client_windows_task resource
Remove the old windows cookbook link. This is EOL and not needed in modern chef
Fix resource names to be lowercase and fix a few typos

### CODE REVIEW

File: content/install_windows.md [hunk 1]
```diff
@@ -20,29 +20,27 @@ aliases = ["/install_windows.html"]
...
 {{% knife_windows_summary %}}
 
 #### Ports
 
 {{% knife_windows_winrm_ports %}}
 
-### Msiexec.exe
+### Use MSI Installer
+
+A Microsoft Installer Package (MSI) is available for installing Chef Infra Client on a Microsoft Windows machine at [Chef Downloads](https://downloads.chef.io/products/infra-client?os=windows)
```
Comment (phiggins) [inline content/install_windows.md]:
```suggestion
A Microsoft Installer Package (MSI) is available for installing Chef Infra Client on a Microsoft Windows machine at [Chef Downloads](https://downloads.chef.io/products/infra-client?os=windows).
```

**Review by phiggins — APPROVED** [review]

File: content/install_windows.md [hunk 2]
```diff
@@ -6,54 +6,41 @@ aliases = ["/install_windows.html"]
...
-configurability, and reliability around log rotation and permissions, as
-compared to running Chef Infra Client as a service. A scheduled task:
+#### Running as a Scheduled Task
 
--   Prevents log rotation issues, such as filling a disk partition
--   Does not have an indeterminate status, such as a service for which
-    only the watcher is "running"
--   Prevents insufficient permissions related to service context
--   Prevents issues related to a system reboot
+On the Microsoft Windows Chef Infra Client should be run periodically as a scheduled task. Scheduled tasks provides visibility, configurability, and reliability around log rotation and permissions. You can configure the Chef Infra Client to run as a scheduled task using the [chef_client_scheduled_task](/resources/chef_client_scheduled_task/) resource.
```
Comment (mwrock) [inline content/install_windows.md]: On the Microsoft Windows Chef Infra Client should be run periodically as a scheduled task. -> If you need to run the Chef Infra Client periodically on Microsoft Windows,  you should use a scheduled task.

Comment (mwrock) [reply content/install_windows.md]: provides -> provide

File: content/windows.md [hunk 3]
```diff
@@ -202,36 +194,37 @@ Two community supports two provisioners for Kitchen:
...
 
--   [Chocolatey_config](/resources/chocolatey_config/)
--   [Chocolatey_package](/resources/chocolatey_package/)
--   [Chocolatey_source](/resources/chocolatey_package/)
+-   [chocolatey_config](/resources/chocolatey_config/)
+-   [chocolatey_package](/resources/chocolatey_package/)
+-   [chocolatey_source](/resources/chocolatey_package/)
+-   [chef_client_scheduled_task](/resources/chef_client_scheduled_task/)
 -   [dsc_resource](/resources/dsc_resource/)
--   [resource_registry_key](/resources/registry_key/)
```
Comment (mwrock) [inline content/windows.md:209]: Isn't it just `registry_key`?

**Review by mwrock — CHANGES_REQUESTED** [review]

File: content/install_windows.md (hunk 2 above)
Comment (tas50) [reply content/install_windows.md]: There's def an extra word here, but I don't think we should make it seem like it's an option to run the client periodically.

Comment (mwrock) [reply content/install_windows.md]: Oh ok that's fair.

File: content/windows.md (hunk 3 above)
Comment (tas50) [reply content/windows.md:209]: yep I fixed that one

**Review by mwrock — APPROVED** [review]

File: content/install_windows.md [hunk 4]
```diff
@@ -6,54 +6,41 @@ aliases = ["/install_windows.html"]
...
     weight = 20
 +++
 
 [\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/install_windows.md)
 
-## Nodes
+## Installation Methods
 
-{{% node %}}
+There are several methods available to install Chef Infra Client, depending on the needs of your organization
```
Comment (IanMadd) [inline content/install_windows.md]:
```suggestion
There are several methods available to install Chef Infra Client depending on the needs of your organization.
```

File: content/install_windows.md [hunk 5]
```diff
@@ -6,54 +6,41 @@ aliases = ["/install_windows.html"]
...
-configurability, and reliability around log rotation and permissions, as
-compared to running Chef Infra Client as a service. A scheduled task:
+#### Running as a Scheduled Task
 
--   Prevents log rotation issues, such as filling a disk partition
--   Does not have an indeterminate status, such as a service for which
-    only the watcher is "running"
--   Prevents insufficient permissions related to service context
--   Prevents issues related to a system reboot
+On Microsoft Windows Chef Infra Client should be run periodically as a scheduled task. Scheduled tasks provides visibility, configurability, and reliability around log rotation and permissions. You can configure the Chef Infra Client to run as a scheduled task using the [chef_client_scheduled_task](/resources/chef_client_scheduled_task/) resource.
```
Comment (IanMadd) [inline content/install_windows.md]:
```suggestion
On Microsoft Windows, run Chef Infra Client periodically as a scheduled task. Scheduled tasks provides visibility, configurability, and reliability around log rotation and permissions. You can configure the Chef Infra Client to run as a scheduled task using the [chef_client_scheduled_task](/resources/chef_client_scheduled_task/) resource.
```

Comment (tas50) [reply content/install_windows.md]: This sentence feels really awkward now.

Comment (IanMadd) [reply content/install_windows.md]: Could just be 

On Microsoft Windows, run Chef Infra Client periodically as a scheduled task.

**Review by IanMadd — APPROVED** [review]

**Comment by phiggins** [comment]
There's a DCO signoff missing here.

**Comment by tas50** [comment]
DCO fixed

**Pull request merged**